community/openjdk7/APKBUILD

```diff
@@ -20,12 +21,12 @@ makedepends="bash findutils tar zip file paxmark gawk util-linux libxslt
 	lcms2-dev libxp-dev libxtst-dev libxinerama-dev zlib-dev
 	libxrender-dev alsa-lib-dev freetype-dev fontconfig-dev
 	gtk+2.0-dev krb5-dev attr-dev pcsc-lite-dev lksctp-tools-dev
-	libxcomposite-dev nss-static"
+	libxcomposite-dev nss-static xz"
 install=""
 
 # upstream binary versions
 RHINO_VER=1.7.7.2
-ANT_VER=1.9.11
+ANT_VER=1.9.14
```

Comment:
Is there any particular reason to increment the antversion too?

Reply:
Is there any particular reason to use the old version? :stuck_out_tongue_winking_eye:

Reply:
Except for the "never change a running (build) system"? None

Reply:
Done.